modules/gentemplates/swagger-ui/src/main/java/com/reprezen/genflow/openapi/swaggerui/v3/XGenerateSwaggerUIv3.xtend

```diff
@@ -182,16 +183,41 @@ class XGenerateSwaggerUIv3 {
 		return jsonMapper.writeValueAsString(tree)
 	}
 
-	private def orNull(Object value) '''«IF value !== null»"«value»"«ELSE»null«ENDIF»'''
-
-	private def maybeFunction(String option) {
-		if (option === null || option.isEmpty) return '''null'''
-
-		// apisSorter and methodsSorter options take either fixed string values (e.g. "alpha") or a Javascript function object. 
-		// The latter should not be quoted, while the former should. This method applies quoting unless the value begins with "function".
-		if(option.startsWith("function")) option else '''"«option»"'''
+	private def asOption(Object value, String key) {
+		switch value {
+			Boolean: '''«key»: «value»,'''
+			Integer: '''«key»: «value»,'''
+			String: '''
+				«IF !value.isNullOrEmpty»
+				«key»: «IF value.startsWith("[") || value.startsWith("{")»«value»«ELSE»"«value»"«ENDIF»,
```

Comment:
I don't understand why we omit quotes on string values that start with opening square bracket or curly brace. Can you add an explanatory comment to the code?

Reply:
Because it's JavaScript not JSON, we want it to be an array, not a string that contains an array.